From 5f9feeee16a26ffebb4ec5fdae26c1565b30ff86 Mon Sep 17 00:00:00 2001
From: Aisheng.Dong <b29396@freescale.com>
Date: Fri, 30 Jul 2010 14:10:21 +0800
Subject: [PATCH] ENGR00125731-1 mxc esdhc: add two DLL control flags in platform_data

Add dll override mode enable and the number of delay cells flag
in platform_data.
This allows user to regulate clock delay line in platform specific code
according to different board and cards.

Signed-off-by: Aisheng.Dong <b29396@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mmc.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mmc.h b/arch/arm/plat-mxc/include/mach/mmc.h
index b23798a..c01dabb 100644
--- a/arch/arm/plat-mxc/include/mach/mmc.h
+++ b/arch/arm/plat-mxc/include/mach/mmc.h
@@ -40,6 +40,8 @@ struct mxc_mmc_platform_data {
 	unsigned int min_clk;
 	unsigned int max_clk;
 	unsigned int clk_flg;	/* 1 clock enable, 0 not */
+	unsigned int dll_override_en;	/* Enable dll override delay line */
+	unsigned int dll_delay_cells;	/* The number of delay cells (0-0x3f) */
 	unsigned int reserved:16;
 	unsigned int card_fixed:1;
 	unsigned int card_inserted_state:1;
-- 
1.5.4.4

